be/node/index.ts

```diff
@@ -23,164 +26,79 @@ app.use(zip());
 app.post(
   "/musicxml-to-svg",
   upload.single("musicxml"),
-  [
-    body("pageWidth").isFloat({ gt: 0 }),
-    body("pageHeight").optional().isFloat({ min: 0 }),
```

Comment:
I'm guessing that you want to use the original PDF for page flipping and use this endpoint for scrolling continuous mode, therefore, we don't really need to have `pageHeight`, right?

I do have a question though. What if the PDF doesn't quite fit nicely with the tablet though? Then, we would have to worry about the music sheet too small if we want to fit the height, or extra scrolling before flipping if we want to fit the width. I think the risk of having the PDF not fitting nicely might increase complexity, but I want to know what everyone else thinks about it.

Reply:
I think we can assume that the PDF pages will each fit nicely on the display. Paged sheet music is meant to be printed and will have similar aspect ratios to tablets.